modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID array using the `mdadm` utility, and integrate the RAID array with {lvms}.
+You can integrate the RAID array with {lvms} while creating the `LVMCluster` custom resource (CR). 
+
+.Prerequisites
+
+* You created a software RAID during the {product-title} installation.
++
+[NOTE]
```

Comment:
Switch to an IMPORTANT admonition as the statement is important. See https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Updated as per the suggestion.

Reply:
:heavy_check_mark: